### PR TITLE
[fix bug 1416764] Update Firefox favicon for Quantum 57

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Updates Firefox favicon behind `firefox57-release` switch
- Removes `media/img/firefox/favicon.png` (orphaned, afaict)

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1416764

## Testing
- Make sure new favicon shows correctly. 
- Make sure I didn't miss anything?

### CODE REVIEW

**Comment by alexgibson** [comment]
Open question: is the switch here worth it? I could be persuaded it's overkill

**Comment by alexgibson** [comment]
Looks like I may need to update the `.ico` file to support more sizes

**Comment by alexgibson** [comment]
OK updated! (I made sure the old .ico shows up in IE 7/8)

File: bedrock/firefox/templates/firefox/base-pebbles.html [hunk 1]
```diff
@@ -4,18 +4,42 @@
...
 {% else %}
   {{ static('img/firefox/template/page-image.png') }}
 {% endif %}
 {% endblock %}
-{% block page_favicon %}{{ static('img/firefox/favicon.ico') }}{% endblock %}
-{% block page_favicon_large %}{{ static('img/firefox/favicon-196.png') }}{% endblock %}
-{% block page_ios_icon %}{{ static('img/firefox/ios-icon-180.png') }}{% endblock %}
+
+{% block page_favicon %}
+{% if quantum %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/base-pebbles.html]: Hmm, it seems this indentation is adding an extra space before and after the `href`? Should still be ok though?

Comment (jpetto) [reply bedrock/firefox/templates/firefox/base-pebbles.html]: Pretty sure you can fix this with the no-whitespace controls:

`{%- if quantum -%}`

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/base-pebbles.html]: Ah, yes! thanks

**Comment by alexgibson** [comment]
Ok, updated using the no-whitespace controls (thanks @jpetto)

File: bedrock/firefox/templates/firefox/base-pebbles.html (hunk 1 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/base-pebbles.html]: Yeah whitespace *should* be ignored in HTML but it does look much nicer without it.

**Pull request merged**